save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestController.kt

```diff
@@ -17,12 +18,16 @@ class TestController {
     @Autowired
     private lateinit var testService: TestService
 
+    @Autowired
+    private lateinit var testExecutionService: TestExecutionService
+
     /**
-     * @param tests
+     * @param testDtos
      */
     @PostMapping("/initializeTests")
-    fun initializeTests(@RequestBody tests: List<Test>) {
-        testService.saveTests(tests)
+    fun initializeTests(@RequestBody testDtos: List<TestDto>) {
+        val testsId = testService.saveTests(testDtos)
```

Comment:
```suggestion
        val testIds = testService.saveTests(testDtos)
```
This will mark this variable as a collection more clearly